INSTRUMENTS/MOCK/MOCK_WEINSCHEL8320/MOCK_WEINSCHEL8320.py

```diff
@@ -0,0 +1,44 @@
+from flojoy import flojoy, Scalar
+from qcodes.instrument_drivers.weinschel import Weinschel8320
+from typing import Optional
+
+
+@flojoy(deps={"qcodes": "0.39.1", "pyvisa-sim": "0.5.1"})
+def MOCK_WEINSCHEL8320(
+    default: Optional[Scalar] = None,
```

Comment:
I think this could be `DataContainer` (any) instead of restricting it to only `Scalar`